packaging/greenboot/flightctl-agent-running-check.sh

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env bash
+
+set -x -eo pipefail
+
+TIMEOUT=60
+
+# This script is used to check if the flightctl-agent service is running It is
+# used in the greenboot process to ensure that the flightctl-agent service is
+# running before proceeding The script will wait for the service to be active,
+# and will fail if the service is not active within the timeout
+
+verify_flightctl_agent_status() {
```

Comment:
This won't catch a flaky agent, right?  (i.e., keeps starting and crashing)
Not a blocker, but if this is true then you can open a continuation JIRA.

Reply:
I am also unsure how greenboot would handle that as this is a "is boot sane" check failures during runtime might need another way of monitoring I will dig further.

Reply:
I meant something like "make sure it stays up for a minute straight before assuming it's OK".  Of course it can crash again later.